tests/test_utils.py

```diff
@@ -1255,7 +1255,11 @@ def test_detect_travis():
 
     Simply check it passes without exception.
     """
-    _ = utils.env_detection.running_on_travis()
+    on_travis = utils.env_detection.running_on_travis()
+    if "ON_TRAVIS" in os.environ:
+        assert on_travis
+    else:
+        assert not on_travis
```

Comment:
Should be using `TRAVIS` instead of `ON_TRAVIS` here

Reply:
See https://docs.travis-ci.com/user/environment-variables/#default-environment-variables